client/cypress/support/redash-api/index.js

```diff
@@ -19,11 +19,11 @@ export function createQuery(data, shouldPublish = true) {
   }, data);
 
   // eslint-disable-next-line cypress/no-assigning-return-values
-  let request = cy.request('POST', '/api/queries', merged);
+  let request = cy.request('POST', '/api/queries', merged).then(({ body }) => body);
```

Comment:
To be consistent with the `shouldPublish = true` changed it to return the body directly.